scripts/openzfs-tracking.sh

```diff
@@ -84,22 +98,102 @@ while getopts 'hd:e:' OPTION; do
 done
 
 cat << EOF
+<!DOCTYPE html>
 <html>
 <head>
 <title>OpenZFS Tracking</title>
 <meta name="keyword" content="zfs, linux"/>
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<script		  src="https://code.jquery.com/jquery-1.12.4.min.js"
+			  integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+			  crossorigin="anonymous"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css">
+<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+
+<script type="text/javascript">
+\$(document).ready(function() {
+	\$('#maintable').DataTable( {
+		 "lengthMenu": [[50, 100, 500, -1], [50, 100, 500, "All"]],
```

Comment:
I think this should default to "All". I'm not 100% sure if you need it honestly.